src/chrome/content/rules/DMLP.org.xml

```diff
@@ -1,36 +1,23 @@
-
 <!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://citmedialaw.org/ => https://www.citmedialaw.org/: Too many redirects while fetching 'https://www.citmedialaw.org/'
-Fetch error: http://www.citmedialaw.org/ => https://www.citmedialaw.org/: Too many redirects while fetching 'https://www.citmedialaw.org/'
-Fetch error: http://dmlp.org/ => https://www.dmlp.org/: Too many redirects while fetching 'https://www.dmlp.org/'
-Fetch error: http://www.dmlp.org/ => https://www.dmlp.org/: Too many redirects while fetching 'https://www.dmlp.org/'
-
-Disabled by https-everywhere-checker because:
-Fetch error: http://citmedialaw.org/ => https://www.citmedialaw.org/: Cycle detected - URL already encountered: https://www.dmlp.org/
-Fetch error: http://www.citmedialaw.org/ => https://www.citmedialaw.org/: Cycle detected - URL already encountered: https://www.dmlp.org/
-Fetch error: http://dmlp.org/ => https://www.dmlp.org/: Cycle detected - URL already encountered: https://www.dmlp.org/
-Fetch error: http://www.dmlp.org/ => https://www.dmlp.org/: Cycle detected - URL already encountered: https://www.dmlp.org/
 	Problematic subdomains:
 
-		- ^	(mismatched, CN: adam.law.harvard.edu)
+		- ^	(mismatched, CN: www.)
 
 
 	Some pages redirect to http.
 
 -->
-<ruleset name="DMLP.org (partial)" default_off="failed ruleset test">
+<ruleset name="DMLP.org (partial)">
 
 	<target host="citmedialaw.org" />
 	<target host="www.citmedialaw.org" />
 	<target host="dmlp.org" />
 	<target host="www.dmlp.org" />
 
+	<test url="http://dmlp.org/favicon.ico" />
+	<test url="http://www.dmlp.org/favicon.ico" />
 
-	<rule from="^http://(?:www\.)?citmedialaw\.org/"
-		to="https://www.citmedialaw.org/" />
-
-	<rule from="^http://(?:www\.)?dmlp\.org/(?!favicon\.ico|files/|misc/|modules/|sites/)"
-		to="https://www.dmlp.org/" />
+	<rule from="^http://(?:www\.)?([^/]+)/"
+		to="https://www.$1/" />
```

Comment:
You can just use multiple simple redirects here.

Reply:
I used this per https://github.com/EFForg/https-everywhere/issues/18910

If you still want me to change, should I be using four literal or two basic rules here?

Reply:
ping @J0WI

Reply:
I prefer two basic rules like in the example [here](https://github.com/EFForg/https-everywhere/issues/18910#issuecomment-600088525)

Reply:
ping @jayvdb